integration/test/app/basic/simple_test.go

```diff
@@ -61,6 +65,33 @@ func TestAppLifecycle(t *testing.T) {
 		Namespace: namespace,
 	}
 
+	{
+		config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("installing chart operator"))
+
+		var tarballPath string
+		{
+			tarballURL := "https://giantswarm.github.io/giantswarm-catalog/chart-operator-0.9.0.tgz"
```

Comment:
The hardcoded version here is a problem. In the VOO spec the design is to have a label called chart-operator.giantswarm.io/version="control-plane". that always maps to master.

https://github.com/giantswarm/giantswarm/blob/master/specs/versioning-of-operators.md#bootstrap

Can we have a TODO issue to get this straight later?

Reply:
Sure, I added 2 TODOs on the code.